### ISSUE TITLE
Mocking a class containing default parameter function without namespace

### ISSUE THREAD

**Issue by cakecatz** [Codeception/AspectMock#109]
It throw exception `[Go\ParserReflection\ReflectionException] Namespace  was not found in the file ~~~.php`.

``` php
<?php
class TargetClass
{
    public function method($param = false) {}
}
```

``` php
test::double('TargetClass');
```

if set number, array or string to `$param`, it will not throw exception.

**Comment by lisachenko** [comment]
Created an issue https://github.com/goaop/parser-reflection/issues/39, should be fixed for 1.1.0 of goaop/parser-reflection soon.

**Comment by lisachenko** [comment]
@cakecatz Please, try to update `goaop/parser-reflection` packet to the latest version ([1.1.0](https://github.com/goaop/parser-reflection/releases/tag/1.1.0)). This should solve this issue.
